<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MyLoginForm {
background: gray;
}


#mainFrame {
border: 3px solid gray;
border-radius: 40px;
background: white;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="STATISTIQUE">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}


QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 50px;
max-width: 50px;
min-height: 13px;
max-height: 13px;
}
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>AJOUT ET MODIFICATION</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-weight: bold;
font-size: 15px;
}</string>
      </property>
      <property name="text">
       <string>immatricule:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-weight: bold;
font-size: 15px;
}</string>
      </property>
      <property name="text">
       <string>marque:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-weight: bold;
font-size: 15px;
}</string>
      </property>
      <property name="text">
       <string>modele:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-weight: bold;
font-size: 15px;
}</string>
      </property>
      <property name="text">
       <string>kilometrage:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-weight: bold;
font-size: 15px;
}</string>
      </property>
      <property name="text">
       <string>maintenance:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-weight: bold;
font-size: 15px;
}</string>
      </property>
      <property name="text">
       <string>capabilite:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_immatricule">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_marque">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_modele">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_kilo">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_maintenance">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_capa">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>320</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>120</y>
        <width>62</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>180</y>
        <width>62</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>AFFICHAGE</string>
     </attribute>
     <widget class="QTableView" name="tab_vehicule">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>591</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="combobox_tri">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>92</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>par defaut</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>immatricule</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>marque</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>modele</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>kilometrage</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>SUPPRESSION</string>
     </attribute>
     <widget class="QPushButton" name="pb_supp">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>62</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_immatricule_supp">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-weight: bold;
font-size: 15px;
}</string>
      </property>
      <property name="text">
       <string>immatricule</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>RECHERCHE</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>31</y>
        <width>641</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_recherche">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>62</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>RECHERCHER</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>IMMATRICULE:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_immatricule_rech">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tab_vehicule_rech">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_pdf">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>350</y>
        <width>62</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>STATISTIQUE</string>
     </attribute>
     <widget class="QLabel" name="stat">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>901</width>
        <height>501</height>
       </rect>
      </property>
      <property name="text">
       <string>l</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>MAINTENANCE</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>IMMATRICULE :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="l_immatricule_Maintenance">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_Maintenance">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>130</y>
        <width>62</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ETAT</string>
      </property>
     </widget>
     <widget class="QLabel" name="resultat_Maintenance">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>230</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>LA RESULTAT DU MAINTENANCE EST :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>CAPABILITE</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
